packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json

```diff
@@ -1,7 +1,7 @@
 {
   "user_account_identity_documents": "Mes documents d'identité",
   "user_account_identity_documents_sub_title": "Déposez vos documents",
-  "user_account_identity_documents_intro": "Vous avez créé votre compte et vous souhaitez faire une commande depuis notre site Indien. La règlementation en vigueur nécessite une vérification de votre identité avant toute commande.",
+  "user_account_identity_documents_intro": "Vous avez créé votre compte et vous souhaitez faire une commande depuis notre site Indien. La règlementation en vigueur nécessite une vérification de votre identité avant toute commande.<br />Assurez-vous que vos données personnelles correspondent au nom et à l'adresse figurant dans vos documents officiels.",
```

Comment:
**nitpick:** Use a different key value pair in translation and add a break-line character in HTML.